plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java

```diff
@@ -89,6 +97,10 @@ public ChannelMavenArtifactRepositoryManager(List<ChannelConfiguration> channels
         this.system = system;
     }
 
+    public ChannelSession getChannelSession() {
```

Comment:
Does this need to be public?

Reply:
This was a left over, I removed the method.